src/DIRAC/TransformationSystem/Utilities/ReplicationTransformation.py

```diff
@@ -1,8 +1,46 @@
 """
 Utilities to create replication transformations
 """
-from DIRAC.TransformationSystem.Client.Transformation import Transformation
 from DIRAC import gLogger, S_OK, S_ERROR
+from DIRAC.TransformationSystem.Client.Transformation import Transformation
+from DIRAC.TransformationSystem.Client.TransformationClient import TransformationClient
+
+
+def getTransformationName(transName, unique=True):
+    """
+    Get a transformation name from a base name
+    If unique is requested, return None if already exists
+    """
+    tName = transName
+    trial = 0
+    transClient = TransformationClient()
+    while True:
+        # Check if there is already a transformation with that name
+        res = transClient.getTransformation(tName)
+        if res["OK"]:
```

Comment:
```suggestion
        if not res["OK"]:
            # transformation does not exist, OK
            return tName
```
no else needed far below